Patches/Core/ThingDefs_Misc/Weapons_Guns.xml

```diff
@@ -843,7 +843,7 @@
           </capacities>
           <power>10</power>
           <cooldownTime>1.9</cooldownTime>
```

Comment:
All of these cooldowns should be changed to 2.44 as well to match the other "heavy weapon" melee tools.

Reply:
Done.